src/base/utils/misc.cpp

```diff
@@ -503,9 +503,9 @@ QList<bool> Utils::Misc::boolListfromStringList(const QStringList &l)
 
 bool Utils::Misc::isUrl(const QString &s)
 {
-    const QString scheme = QUrl(s).scheme();
-    QRegExp is_url("http[s]?|ftp", Qt::CaseInsensitive);
-    return is_url.exactMatch(scheme);
+    static const QRegExp isURL("http[s]?|ftp", Qt::CaseInsensitive);
```

Comment:
I just made it to be `static const` since it is unchangeable inside method and between its calls.
This is a simple change (like changing the case of characters in a variable name). I don't want to be distracted by outside things in this big PR. So if something is wrong with it, I'll just take it as it was.